### ISSUE TITLE
Allow more advanced label size and display settings

### ISSUE THREAD

**Issue by AVermeij** [almende/vis#247]
Hi,

Currently, labels are sized based on fixed `fontSize` properties of nodes. 

It would be worthwhile to allow for an extra option which sizes labels proportionally to the size/radius of the underlying node. This can serve as a great visual aid in networks which are made up of nodes of varying size (e.g. based on their amount of connections). This could for instance be defined by setting a ratio between node size and label size.

Another idea relates to thresholds for displaying labels. It would be interesting to have an option which sets the minimum size which a node needs to have to have its label displayed. At various zoom levels, this then allows better visual insight into e.g. key nodes without attracting attention from peripheral nodes. Labels of such peripheral nodes can then simply be displayed by furthering zooming in to their location.

What's your view on this?

Cheers,

André

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
